compass/src/main/java/com/marfeel/compass/storage/Storage.kt

```diff
@@ -14,17 +14,18 @@ import com.marfeel.compass.core.model.compass.currentTimeStampInSeconds
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
 import java.util.*
 import kotlin.coroutines.CoroutineContext
 
-
 internal class Storage(
 	private val context: Context,
 	coroutineContext: CoroutineContext
 ) {
 	companion object {
 		private const val encryptedStorageName = "EncryptedStorage"
 		private const val fallbackStorageName = "FallbackStorage"
+		private const val rawStorageName = "RawStorage"
```

Comment:
just for being "clean". If we're going to go for everybody with plan prefrences I was feeling really disgusted using a file called FallbackStorage